tm2/pkg/crypto/keys/client/sign.go

```diff
@@ -157,6 +157,24 @@ func execSign(cfg *SignCfg, args []string, io commands.IO) error {
 		}
 	}
 
+	// Check if accountNumber or sequence is zero to determine if account information needs to be fetched
+	if cfg.AccountNumber == 0 || cfg.Sequence == 0 {
```

Comment:
i don't think that we should fetch without using an explicit flag or command.

Reply:
565a4739dc0d74c51f7ab0e35bae87c53d5f33ed